cylc/flow/print_tree.py

```diff
@@ -33,7 +33,8 @@
 
 
 def print_tree(tree, padding, use_unicode=False, prefix='', labels=None,
```

Comment:
Would this be better named now as `build_tree`, to properly reflect what it does now?